src/lib/tsc.ts

```diff
@@ -345,6 +345,7 @@ export function findModulesThatImportModule(project: TSM.Project, id: string) {
   const data: ModulesWithImportSearchResult[] = []
 
   for (const sourceFile of project.getSourceFiles()) {
+    // TODO: if project folder contains "node_modules" in it, nexus modules won't be found
```

Comment:
How so?

Reply:
`projects/my_super_node_modules_bundler/api/index.ts`

Reply:
Ah of course, thanks